### DISCUSSION TITLE
Evidence of Strawberry (o1-preview) intelligence

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/Raspberry#22] [Off Topic]
First post https://x.com/LechMazur/status/1834728714124341581

![image](https://github.com/user-attachments/assets/be3011ac-7278-443f-a179-179301c2e8b1)

**Comment by daveshap** [comment]
https://x.com/maximlott/status/1834652893229859212


![image](https://github.com/user-attachments/assets/20e84e7d-fa44-48b1-861f-24e1ef5e3f4e)

**Comment by daveshap** [comment]
Quality vs price

https://x.com/rohanpaul_ai/status/1834958662416183457

![image](https://github.com/user-attachments/assets/c9d60d91-e15e-48b4-bc18-9b56bc96310b)

**Comment by daveshap** [comment]
![image](https://github.com/user-attachments/assets/ba26bb99-ebb2-4641-b747-f1a3a387a3c5)

**Comment by daveshap** [comment]
It looks like the IQ of o1-preview is converging on about 120, based on different tests I've seen around. 

https://x.com/maximlott/status/1835043371339202639

![image](https://github.com/user-attachments/assets/293d09a1-6f9a-400f-8311-35f7e96ad603)

**Reply by Pils10** [reply]
The difference suggests model familiarity. Most likely is, that the model had picked up one or two guides or sample questions on how to solve these types of challenges and applied them there. From my own personal testing, this model is definitely not dumb, but definitely not above average in intelligence (but it is in knowledge).

**Comment by daveshap** [comment]
https://www.reddit.com/r/singularity/comments/1fh22ur/chatgpt_o1preview_solves_unique_phdlevel/?share_id=2dNRfcjZskIUtXIIfYmDc&utm_content=1&utm_medium=ios_app&utm_name=ioscss&utm_source=share&utm_term=1

![image](https://github.com/user-attachments/assets/0dc15a66-750a-45f3-b9a5-fdcb002c71c1)

**Comment by daveshap** [comment]
https://www.reddit.com/r/singularity/comments/1fhi59o/chatgpt_o1_preview_mini_wrote_my_phd_code_in_1/?share_id=aqPwdl5lztR4GJIPsUpvq&utm_content=1&utm_medium=ios_app&utm_name=ioscss&utm_source=share&utm_term=1

![image](https://github.com/user-attachments/assets/7147c0d6-38bf-43b3-aa2f-10c502d5a93d)
